pyaedt/modeler/cad/Primitives.py

```diff
@@ -8469,6 +8472,43 @@ def _create_cylinder_instance(self, name, cs, origin, data):
 
         return cyl1
 
+    def _create_box_instance(self, name, cs, origin, data):
+        """Create cylinder instance.
```

Comment:
Should this line be deleted and the "Create a box instance." line be used as the brief description?